style.css

```diff
@@ -68,3 +63,249 @@ li {
+}
+
+.short_desc * {
+  margin-right: 10px;
+}
+
+.short_desc h4 {
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.short_desc h4:nth-child(1) {
+  color: #344563;
+  text-transform: uppercase;
+}
+
+.short_desc span {
+  background-color: #c1c7d0;
+  width: 10px;
+  height: 10px;
+  border-radius: 50%;
+}
+
+.work__lang {
+  /* display: flex; */
+  float: left;
+}
+
+.work__lang * {
+  margin: 20px 10px 20px 0;
+  background: #ebebff;
+  color: #6070ff;
+  font-weight: 500;
+  border-radius: 7px;
+  padding: 5px;
+  float: left;
+}
+
+.lang-1:hover,
+.lang-2:hover,
+.lang-3:hover {
+  text-decoration: underline;
+}
+
+.btn {
+  width: 150px;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  outline: none;
+  padding: 12px;
+  border-radius: 8px;
+  color: #6070ff;
+}
+
+.btn:hover {
+  color: #fff;
+  background-color: #6070ff;
+  text-decoration: underline;
+}
+
+.btn:active {
+  color: #fff;
+  background-color: #2230d2;
+}
+
+.btn-enabled {
+  background-color: #fff;
+  border: 1px solid #7f8cff;
+}
+
+/* ABOUT ME SESTION STYLES */
+
+.about-me {
+  display: grid;
+  grid-template-columns: 1fr;
+  width: 327px;
+  height: 1000px;
+  padding: 16px;
+  background: #fff;
+  border-radius: 16px;
+  margin: 50px auto;
+  border-top-right-radius: 50px;
+}
+
+.about-me__title {
+  width: 200px;
+  height: 101px;
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: 700;
+  margin: 20px 0 15px 0;
+}
+
+.about-me-container ul {
+  margin-bottom: 40px;
+}
+
+.about-me-container > a {
+  display: block;
+  width: 160px;
+  margin-top: 15px;
+  padding: 10px;
+}
+
+.about-me__tools {
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.tools {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 10px 0;
+  text-transform: capitalize;
+}
+
+.programming-lang {
+  display: flex;
+  align-items: center;
+  background-color: #f7f7f9;
+  border-radius: 8px;
+  padding: 12px;
+  margin-bottom: 10px;
+  margin-top: 10px;
+  height: 66.73px;
+  width: 303px;
+}
+
+.programming-lang img {
+  width: 48px;
+  height: 48px;
+  margin-right: 12px;
+}
+
+.frameworks,
+.skils {
+  border-bottom: 2px solid #dfe1e6;
+}
+
+/* CONTACT FORM STYLES */
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 779px;
+  padding: 16px;
+  background: #fff;
+  border-radius: 8px;
+  margin: 0 auto;
+  background-image: url("/img/contact-form-hapes.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-color: #6070ff;
+  border-top-left-radius: 50px;
+  margin-top: 20px;
+}
+
+.contact-form h2 {
+  color: #fff;
+  font-weight: 700;
+  font-size: 30px;
+  line-height: 50px;
+  text-align: center;
+  margin-top: 135px;
+}
+
+.contact-form p {
+  color: #fff;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  margin-bottom: 50px;
+}
+
+.form-control {
```

Comment:
- [ ] It would be better to use flexbox for your form to lay your inputs out in one direction. As you layout your inputs you want to control the dimensions of the inputs in that one dimension or control the spacing between inputs.